src/Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -190,6 +190,26 @@
                         <FontIcon FontFamily="{StaticResource RecycleBinIcons}" Glyph="&#xEF88;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
```

Comment:
Can you add a separator that's shows when the extract button is visible?

Reply:
Yep, I'll do that and fix the merge conflicts.